### ISSUE TITLE
Iaentities issue + texture pack reloading on 1.20.3+

### ISSUE THREAD

**Issue by LuluxClient** [PluginBugs/Issues-ItemsAdder#3224]
### Terms

- [X] I'm using the very latest version of ItemsAdder and its dependencies.
- [X] I am sure this is a bug and it is not caused by a misconfiguration or by another plugin.
- [X] I already searched on this [Github page](https://github.com/PluginBugs/Issues-ItemsAdder/issues) to check if the same issue was already reported.
- [X] I already searched on the [plugin wiki](https://itemsadder.devs.beer/) to know if a solution is already known.
- [X] I already searched on the [forums](https://forum.devs.beer/) to check if anyone already has a solution for this.

### Discord tag (optional)

xdLulux

### What happened?

I wanted to support 1.20.4 so I added the dev build of velocity, added viaversion on the spigot side and updated protocollib, lonelibs, itemsadder & nbtapi and now all my iaentities are no longer working, when switching servers on a 1.20.4 client I'm also having an issue where the pack is reloading again and it just glitches out. 

Here are the errors & videos:
- Errors: https://pastebin.com/mp7f61qL
- Video: https://gyazo.com/d54b4172b7e89bff90f95e2e9862541d

More info:
- Velocity: 3.3.0-SNAPSHOT Build # 307
- Citizens: Citizens-2.0.32-SNAPSHOT (build 3188)
- BungeePackFix 1.14

### Steps to reproduce the issue

1. have a 1.20.1 server Pufferfish server
2. upgrade itemsadder from the version ItemsAdder version 3.6.1-beta-r2 to ItemsAdder version 3.6.2d-beta-2
3. Restart and check that citizens iaentites aren't working
4. 1.20.3 and above will have the texture pack reloading issue when it was supposed to be fixed on 1.20.3

### Server version

This server is running Pufferfish version git-Pufferfish-"979d3a2" (MC: 1.20.1) (Implementing API version 1.20.1-R0.1-SNAPSHOT) (Git: 979d3a2 on HEAD)
You are running 7 versions beyond. Please update your server when possible to maintain stability, security, and receive the latest optimizations.

### ItemsAdder Version

version 3.6.2d-beta-2

### ProtocolLib Version

5.2.0 b679

### LoneLibs Version

1.0.40

### FULL server log

private

### Error (optional)

```shell
- Errors: https://pastebin.com/mp7f61qL
- Video: https://gyazo.com/d54b4172b7e89bff90f95e2e9862541d
```

### Screenshots/Videos (you can drag and drop files or paste links)

- Video: https://gyazo.com/d54b4172b7e89bff90f95e2e9862541d

**Comment by LoneDev6** [comment]
Resourcepack issue should be related to this: https://github.com/LoneDev6/BungeePackFix/issues/44

**Comment by LoneDev6** [comment]
The entities issue will be fixed and it's now tracked there #3232, since it's not good to have two issues in the same report, thanks!
